flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/optimization/GradientDescent.scala

```diff
@@ -36,19 +36,20 @@ import org.apache.flink.ml.optimization.Solver._
   * At the moment, the whole partition is used for SGD, making it effectively a batch gradient
   * descent. Once a sampling operator has been introduced, the algorithm can be optimized
   *
-  * @param runParameters The parameters to tune the algorithm. Currently these include:
-  *                      [[Solver.LossFunction]] for the loss function to be used,
-  *                      [[Solver.RegularizationType]] for the type of regularization,
-  *                      [[Solver.RegularizationParameter]] for the regularization parameter,
+  *  The parameters to tune the algorithm are:
+  *                      [[Solver.LossFunctionParameter]] for the loss function to be used,
+  *                      [[Solver.RegularizationTypeParameter]] for the type of regularization,
+  *                      [[Solver.RegularizationValueParameter]] for the regularization parameter,
```

Comment:
I'm doing this to avoid confusion between the LossFunction class and the parameter setting here. Having the same name I think creates some confusion, especially when auto-completing stuff in Intellij. However the names are starting to get pretty long.

You think we should remove this?